.gitignore

```diff
@@ -20,3 +20,13 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+# Gradle
+.gradle
+build
+
+# IntelliJ
+.idea
+*.iml
+
+
```

Comment:
I notice you don't exclude the idea "out" directory, do you configure idea to put it's build output somewhere else?

Reply:
I didn't see it show up so maybe I'm not doing something right in my ide.  I think I built this project from command line more than within the ide.

I'll make the change.